### PR TITLE
MB-1811 Table Elements

### PR DESCRIPTION
**Pull request by Shkeating**
## Description
this branch sets up the styling and class names for all the various table cell styles that come up in the table-based components in our system.

## Reviewer Notes

- I am aware the chevron right and left are facing the same direction. any tips on how to resolve that would be appreciated!
- currently I am implementing pagination with `.usa-select` which I am not convicned is the best approach. is there a USWDS component specifically for numeric fields with this styling? should I just make a new class and `@extend` `.usa-select`'s styling?

## Setup

- make sure to run a `yarn install` if you haven't recently
- make sure Docker is running

```sh
make storybook
```

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1811?atlOrigin=eyJpIjoiZTBjOTM0N2U5ZTIxNDFjMzgzMGQwZjMxMmI4YzJhMDEiLCJwIjoiaiJ9) for this change
* [Abstract](https://dp3.atlassian.net/browse/MB-1811?atlOrigin=eyJpIjoiZTBjOTM0N2U5ZTIxNDFjMzgzMGQwZjMxMmI4YzJhMDEiLCJwIjoiaiJ9) shows the cell styles that we've added in this branch

## Screenshots

image

### CODE REVIEW

File: src/stories/index.stories.jsx [hunk 1]
```diff
@@ -325,3 +327,244 @@ storiesOf('Components|Containers', module).add('all', () => (
...
+        </div>
+        <div className="sb-table-wrapper">
+          <code>td-numeric</code>
+          <table>
+            <tbody>
+              <tr>
+                <td>Table Cell Content</td>
+              </tr>
+            </tbody>
+          </table>
```
Comment (gidjin) [inline src/stories/index.stories.jsx:377]: should the `<td>` on line 374 have a class of `numeric`?

File: src/stories/index.stories.jsx [hunk 2]
```diff
@@ -325,3 +327,244 @@ storiesOf('Components|Containers', module).add('all', () => (
...
+        </div>
+        <div className="sb-table-wrapper">
+          <code>th—numeric</code>
+          <table>
+            <thead>
+              <tr>
+                <th>Table Cell Content</th>
+              </tr>
+            </thead>
+          </table>
```
Comment (gidjin) [inline src/stories/index.stories.jsx:417]: same question here about the class for the `th`

File: src/stories/index.stories.jsx [hunk 3]
```diff
@@ -325,3 +327,244 @@ storiesOf('Components|Containers', module).add('all', () => (
     </div>
   </div>
 ));
+
+// Tables
```
Comment (gidjin) [inline src/stories/index.stories.jsx:331]: The index file is getting long. I'd recommend creating a new file for table stories. As long as it's name ends in `.stories.jsx` storybook will pick it up. Maybe something like `tables.stories.jsx`

File: src/stories/index.stories.jsx (hunk 1 above)
Comment (Shkeating) [reply src/stories/index.stories.jsx]: yes. done!

File: src/stories/index.stories.jsx (hunk 2 above)
Comment (Shkeating) [reply src/stories/index.stories.jsx]: yup!

**Pull request merged**